manifests/init.pp

```diff
@@ -278,6 +278,8 @@
 #
 # $dhcp_load_balance::          Cutoff after which load balancing is disabled
 #
+# $dhcp_manage_acls::           If true, will manage ACLs on RedHat systems
```

Comment:
Whether to manage DHCP directory ACLs. This allows the Foreman Proxy user to access even if the directory mode is 0750.